### ISSUE TITLE
Reference potential not echoed in FitSNAP output

### ISSUE THREAD

**Issue by athomps** [FitSNAP/FitSNAP#168]
FitSNAP does not provide a definition of the reference potential as output.  This is a serious weakness, since it requires human intervention to manually convert whatever is in [REFERENCE] e.g.

```
pair_style = hybrid/overlay zero 10.0 zbl 4.0 4.8
pair_coeff1 = * * zero
pair_coeff2 = * * zbl 73 73
```

to 

```
pair_style = hybrid/overlay zbl 4.0 4.8 snap
pair_coeff * * zbl 73 73
pair_coeff * * snap
```

This is both time-consuming and very prone to error, as recent history has shown.

The solution is simple. Take the already existing information in `config.sections["REFERENCE"].lmp_pairdec` and echo it to another file called e.g. `*_pot.snap`.  This will both make life easier for users and avoid preventable errors.

**Comment by mitwood** [comment]
I agree with @athomps, but we should supply an example LAMMPS input file, not append the pair-style definitions to the param file. The code already generates its own input file when calculating descriptors, can adapt for this use too.

**Comment by rohskopf** [comment]
@mitwood full LAMMPS input requires atom positions/geometry which is hard to assume for users, so maybe potential file is sufficient. We could supply a template LAMMPS input that loads the potential file, but users can't run the template LAMMPS input unless they supply atom positions.

**Comment by rohskopf** [comment]
See new draft PR. I'm writing a potential file called `*_pot.mod` which looks like this:

```
pair_style hybrid/overlay zbl 4.0 4.8 snap
pair_coeff * * zbl 73 73
pair_coeff * * Ta_pot.snapcoeff Ta_pot.snapparam Ta
```

Then we use this in a template LAMMPS input script called `in.lammps` like this:

```
# LAMMPS template input written by FitSNAP.
# Runs a NVE simulation at specified temperature and timestep.

# Declare simulation variables

variable timestep equal 0.5e-3
variable temperature equal 600

units metal

# Supply your own data file below

read_data DATA

# Include potential file

include Ta_pot.mod

# Declare simulation settings

timestep ${timestep}
neighbor 1.0 bin
velocity all create ${temperature} 10101 rot yes mom yes
fix 1 all nve

# Run dynamics

run 1000
```

**Comment by mitwood** [comment]
Regarding atom positions, we just take a training file at random (or the first one). The calculator classes generate this in a LAMMPS file through a series of 'create_atoms single' and 'set' commands.

**Comment by mitwood** [comment]
But the template looks nice. Make sure it keeps some 'designed flaws' like the data file being missing so it won't run out of the box if you want to stick with the template idea.

**Comment by athomps** [comment]
Writing out the *_pot.mod is great. That eliminates much of the preventable error risk.  There are still some subtleties with LAMMPS types in the reference potential, where the user may wish to have a different set of types in their application, in which case they will have to modify *_pot.mod. But this is something that could be explained in documentation and/or in the LAMMPS input file. The idea of providing a working LAMMPS input file is a good one. This could also be used for regression testing and just for general sanity checking.

**Comment by rohskopf** [comment]
Fixed in #170
